Code/Framework/AzQtComponents/AzQtComponents/Components/StyleManager.cpp

```diff
@@ -47,9 +51,43 @@ namespace AzQtComponents
     static QStyle* createBaseStyle()
     {
         return QStyleFactory::create("Fusion");
-
     }
 
+    bool StyleManager::IsStylePropertyDefined(const char* propertyKey) const
+    {
+        return (m_themeProperties.contains(propertyKey));
+    };
+
+    QString StyleManager::GetStylePropertyAsString(const char* propertyKey) const
+    {
+        if (m_themeProperties.contains(propertyKey))
+        {
+            return m_themeProperties.value(propertyKey);
+        }
+
+        return QString();
+    };
+
+    int StyleManager::GetStylePropertyAsInteger(const char* propertyKey) const
+    {
+        if (m_themeProperties.contains(propertyKey))
+        {
+            return m_themeProperties.value(propertyKey).toInt();
+        }
+
+        return int();
+    };
+
+    QColor StyleManager::GetStylePropertyAsColor(const char* propertyKey) const
+    {
+        if (m_themeProperties.contains(propertyKey))
+        {
+            return QColor(m_themeProperties.value(propertyKey));
+        }
+
+        return QColor();
+    };
+
```

Comment:
Instead of doing the look up twice by calling contains and then value, you should be able to use find to get the iterator and return the value if it's a valid iterator. 

QHash has an overloaded version of the value function that allows you to pass in a default value to return if the value is not in the container. That will collapse all of these functions down to one line.